tests/unit/test_hooks.py

```diff
@@ -149,6 +150,12 @@ def identity_hook(z, hook):
     model.set_use_split_qkv_input(True)
     model.add_hook("blocks.0.hook_q_input", identity_hook)
 
+    model.reset_hooks()
+    model.set_use_attn_in(True)
+    model.add_hook("blocks.0.hook_attn_in", identity_hook)
+
```

Comment:
Since the shape should be the same if split_qkv_input is True OR if set_use_attn_in is True, it might be worth a separate test here - my understanding is that this test would pass even if set_use_attn_in did nothing.